swatch-system-conduit/build.gradle

```diff
@@ -18,6 +18,7 @@ openApiGenerate {
         interfaceOnly: "true",
         generatePom: "false",
         dateLibrary: "java8",
+        useTags: "true"
```

Comment:
What's this setting do?

Reply:
It makes it so that the tag attributes in your spec are used as class names when you generate them.